tests/test_rut_crypto_utils.py

```diff
@@ -0,0 +1,56 @@
+import unittest
+from unittest.mock import Mock, patch
+
+from cryptography.hazmat.backends.openssl import backend as crypto_x509_backend
+
+from cl_sii import rut  # noqa: F401
+from cl_sii.libs.crypto_utils import load_der_x509_cert
+from cl_sii.rut.crypto_utils import get_subject_rut_from_certificate_pfx
+
+from . import utils
+
+
+class FunctionsTest(unittest.TestCase):
+
+    def test_get_subject_rut_from_certificate_pfx_ok(self) -> None:
+        cert_der_bytes = utils.read_test_file_bytes(
+            'test_data/sii-crypto/DTE--76354771-K--33--170-cert.der')
```

Comment:
Let's format this with *Black*.

```suggestion
            'test_data/sii-crypto/DTE--76354771-K--33--170-cert.der'
        )
```

Reply:
🖤